mod-clans/src/test/java/gg/solarmc/loader/clans/test/ClanIT.java

```diff
@@ -68,6 +68,49 @@ private void assertMembers(Set<ClanMember> members, Clan clan) {
         assertEquals(members, clan.currentMembers(), "Cached members updated incorrectly");
     }
 
+    @Test
+    public void setClanLeader() {
+        OnlineSolarPlayer leader = dataCenterInfo.loginNewRandomUser();
+        OnlineSolarPlayer usurper = dataCenterInfo.loginNewRandomUser();
```

Comment:
Good variable name!

Reply:
i saw you making witty names and thought i'd continue the tradition